packages/rest/src/writer.ts

```diff
@@ -6,6 +6,7 @@
 import {OperationRetval, Response} from './types';
 import {HttpError} from 'http-errors';
 import {Readable} from 'stream';
+const buildResponseData = require('strong-error-handler').buildResponseData;
```

Comment:
Let's add a `.d.ts` file to strong-error-handler so that we can use it in a type-safe way here.

```ts
import {buildResponseData} from 'strong-error-handler';
```